addons/survey/data/survey_demo_conditional.xml

```diff
@@ -14,7 +14,7 @@
         <field name="questions_layout">page_per_question</field>
         <field name="description" type="html">
             <p>Choose your favourite subject and show how good you are. Ready?</p></field>
-        <field name="background_image" type="base64" file="survey/static/src/img/burger_quiz_background.jpg"/>
+        <field name="background_image" type="base64" file="survey/static/src/img/burger_quiz_background.webp"/>
```

Comment:
Note that it is 126 Kb, which violates the Antony's rule ("no more than 10 Kb, here is a trick to reduce the size [disappears]"). I understand a neat background is necessary, but can't we reduce it a bit ?

Reply:
Note that it is an improvement over the previous image which weighed 356kB :grimacing: We'll see if we can save some more bytes, e.g. by lowering the resolution.

Reply:
We kept the original image but we reduced it to 40kB, reducing it even further drastically reduces the quality

Reply:
Cool :D thanks for saving bandwidth !